CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Changelog for Critical Maps iOS
 - Fix: NavigationBar Colors under iOS 13 
 - Fix: UITableViewHeaderFooterView backgroundColor deprecation warning
 - Fix: Ambiguous auto layout constraints for Settings screen
+- Fix: Don't update content when slightly swipe down in Social Modal
```

Comment:
I don't know if this is the correct line for this or if I should put it under a new version on top?

Reply:
cc @headione

Reply:
I would say you can open an Unreleased section since a new version will be > 3.5, right?!
https://keepachangelog.com/en/1.0.0/

Reply:
Okay, done ✅